static/styles/left-sidebar.scss

```diff
@@ -132,18 +131,18 @@ li.active-sub-filter {
 
 #global_filters .global-filter {
     position: relative;
-    padding: 1px 10px;
+    padding: 1px 5px;
 }
 
 .left-sidebar li {
-    padding-top: 2px;
-    padding-bottom: 2px;
+    // padding-top: 2px;
+    // padding-bottom: 2px;
 
     border-radius: 4px;
 }
 
 #global_filters .global-filter i {
-    font-size: 14px;
+    // font-size: 14px;
```

Comment:
What's the story with commenting these out?

Reply:
I think because it was not doing anything anyway. 
But yeah, perhaps it should not be commented out.